### PR TITLE
Bug 1833546: Label the nodes with test labels, and destroy only ptp configs created by tests

### PR DESCRIPTION
**Pull request by fedepaol**
Also, fix the priority test.

### CODE REVIEW

File: test/ptp/ptp.go [hunk 1]
```diff
@@ -262,10 +264,46 @@ var _ = Describe("[ptp]", func() {
 		var testPtpPod v1core.Pod
 		var ptpConfigSlave ptpv1.PtpConfig
 
-		BeforeEach(func() {
-			ptpConfigName := "test"
+		execute.BeforeAll(func() {
```
Comment (SchSeba) [inline test/ptp/ptp.go:267]: Can we use the same one as in the previews context?

I will like to clean both the grandmaster and the slave and recreate them

File: test/ptp/ptp.go [hunk 2]
```diff
@@ -262,10 +264,46 @@ var _ = Describe("[ptp]", func() {
...
+				ptpSlaveNode.InterfaceList[0],
+				"-s",
+				"-a -r",
+				PtpSlaveNodeLabel,
+				pointer.Int64Ptr(5))
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		BeforeEach(func() {
+			response, err := client.Client.CoreV1Interface.RESTClient().Get().AbsPath(ConfigPtpAPIPath + "/" + PtpSlavePolicyName).DoRaw()
```
Comment (SchSeba) [inline test/ptp/ptp.go:306]: we can use the client for this one now, no need for a restClient request

File: test/ptp/ptp.go (hunk 1 above)
Comment (SchSeba) [reply test/ptp/ptp.go:267]: we can put everything from the BeforeAll in a function and call it here and in the other Context section

Comment (fedepaol) [reply test/ptp/ptp.go:267]: Any reason we can't merge the two together then?
The test body would then be wait, delete and check (which I think makes more sense).

**Comment by fedepaol** [comment]
/hold
need to validate the new version

**Comment by fedepaol** [comment]
Validated the new version:
```
[root@fci1-installer ptp-operator]# make functests
hack/run-functests.sh
/root/fede/ginkgo
Running Suite: PTP e2e integration tests
========================================
Random Seed: 1589190200
Will run 8 of 8 specs

•••
------------------------------
• [SLOW TEST:35.671 seconds]
[ptp]
/root/fede/ptp-operator/test/ptp/ptp.go:30
  PTP e2e tests
  /root/fede/ptp-operator/test/ptp/ptp.go:86
    PTP Interfaces discovery
    /root/fede/ptp-operator/test/ptp/ptp.go:172
      The interfaces support ptp can be discovered correctly
      /root/fede/ptp-operator/test/ptp/ptp.go:188
------------------------------
•••
------------------------------
• [SLOW TEST:34.428 seconds]
[ptp]
/root/fede/ptp-operator/test/ptp/ptp.go:30
  PTP e2e tests
  /root/fede/ptp-operator/test/ptp/ptp.go:86
    PTP Interfaces discovery
    /root/fede/ptp-operator/test/ptp/ptp.go:172
      Can provide a profile with higher priority
      /root/fede/ptp-operator/test/ptp/ptp.go:262
------------------------------

Ran 8 of 8 Specs in 84.479 seconds
SUCCESS! -- 8 Passed | 0 Failed | 0 Pending | 0 Skipped
PASS
```

**Pull request merged**